src/test/scala/play/api/cache/redis/util/ExpirationSpec.scala

```diff
@@ -4,6 +4,7 @@ import java.util.Date
 
 import play.api.cache.redis.{CacheApi, Expiration, ExpirationImplicits, Redis}
 
+// INFO JodaDateTime is deprecated as for 2018 use ZonedDateTime or LocalDateTime
```

Comment:
Thanks for your note, I'm aware of it. However, this is a test and then there is pending #108 for tests refactoring, which will include this change.